spec/workers/detect_invariants_daily_check_spec.rb

```diff
@@ -73,28 +73,22 @@
       expect(Sentry).not_to have_received(:capture_exception)
     end
 
-    it 'detects applications submitted with the same course' do
-      allow(Sentry).to receive(:capture_exception).with(an_instance_of(described_class::ApplicationSubmittedWithTheSameCourse))
+    it 'doesn’t alert when applications with the same course have different sites' do
```

Comment:
can we remove the whole test as we no longer have any code to support it?

Reply:
that's deliberate, in that we're testing we've successfully removed the check. but yes, probably best not to have it hanging around in the codebase.